apps/services/auth-public-api/src/app/modules/delegations/meDelegations.controller.ts

```diff
@@ -132,6 +132,14 @@ export class MeDelegationsController {
     type: 'string',
     description: 'Delegation ID.',
   })
+  @ApiQuery({
+    name: 'valid',
```

Comment:
renaming this to `validity` makes more sense IMO ... `valid` almost sounds like a boolean.

Reply:
^ or anything that would describe it better

Reply:
This query argument is already used in other AuthPublicAPI endpoints, which should be consistent. It would be a bigger change to rename it everywhere.

Not sure if this is a big issue, since the argument is documented in swagger as an enum rather than a boolean, and validated properly.